src/AppRoutes.tsx

```diff
@@ -83,7 +83,16 @@ export const AppRoutes: React.FC<AppRoutesProps> = ({
 
         {/* Main Messenging container once you are unlocked */}
         <Route element={<RequireUnlockedWallet />}>
-          <Route path=":walletId" element={<MessengerContainer />} />
+          <Route path=":walletId" element={<MessengerContainer />}>
+            <Route index element={<Navigate to="directs" replace />} />
+            <Route path="directs" element={<MessengerContainer />} />
+            <Route path="directs/:contactId" element={<MessengerContainer />} />
+            <Route path="broadcasts" element={<MessengerContainer />} />
+            <Route
+              path="broadcasts/:channelId"
+              element={<MessengerContainer />}
+            />
+          </Route>
```

Comment:
in // of the comment [here](https://github.com/K-Kluster/Kasia/pull/245/files#r2358496016), we probably will need to properly split responsibilities per router (new component per route)